charts/argo/templates/server-deployment.yaml

```diff
@@ -63,6 +63,14 @@ spec:
             value: {{ .Values.server.baseHref | quote }}
           resources:
             {{- toYaml .Values.server.resources | nindent 12 }}
+          volumeMounts:
+          {{- if .Values.server.volumeMounts }}
+  {{- toYaml .Values.server.volumeMounts | nindent 12}}
```

Comment:
We normally indent in the `toYaml` to where it would sit so things are easier to ready, mind doing that?